### DISCUSSION TITLE
External rbac

### DISCUSSION THREAD

**Discussion by kotyara85** [gravitational/teleport#16584] [General]
Hello everyone,
Does teleport support external rbac? We have a specific case where we want to use OPA for policy purposes
Thanks

**Comment by webvictim** [comment]
Not currently, but I believe we have something on the roadmap which might address this use case.

**Reply by kotyara85** [reply]
Is there any ETA on this and maybe some docs?
Thanks!

**Reply by webvictim** [reply]
I don't have anything more I can share at the moment.

**Reply by webvictim** [reply]
This is the RFD for the "login rules" feature: https://github.com/gravitational/teleport/pull/14977
